administrator/language/de-DE/joomla.ini

```
@@ -92,7 +92,7 @@ JHIDE="Verbergen"
 JHIDEPASSWORD="Passwort ausblenden"
 JHOMEDASHBOARD="Dashboard"
 JINVALID_TOKEN="Die Anfrage wurde zurückgewiesen, da der Sicherheitstoken ungültig ist. Bitte die Seite aktualisieren und erneut versuchen."
-JINVALID_TOKEN_NOTICE="Der Sicherheitstoken ist falsch. Die Anfrage wurde zurückgewiesen, um eine Sicherheitsverletzung zu verhindern. Bitte die Seite aktualisieren und erneut versuchen."
+JINVALID_TOKEN_NOTICE="Der Sicherheitstoken stimmte nicht überein. Die Anfrage wurde abgebrochen, um eine Sicherheitsverletzung zu verhindern. Bitte erneut versuchen."
```

```suggestion
JINVALID_TOKEN_NOTICE="Die Sicherheitstoken stimmen nicht überein. Die Anfrage wurde abgebrochen, um eine Sicherheitsverletzung zu verhindern. Bitte erneut versuchen."
```

Ggfs ist es so flüssiger @tecpromotion ? Müsste dann in allen Stellen angepasst werden